sphinxcontrib/opendataservices.py

```diff
@@ -232,16 +232,17 @@ def run(self):
         return [bl]
 
 
-def parse_markdown(text):
-    return to_docutils(text).children[:]
+def parse_markdown(text, document=None):
```

Comment:
Why is =None here? Looks like it's never called without a value, and wouldn't the next line error anyway if it was None?

Reply:
This default is needed for LocalizationNote below. However, it doesn't seem to actually get called by the example in the docs, so I'm not sure what's going on with that.

Reply:
I've fixed this so that it won't error when document is None. I think whatever's happening with LocalizationNote was broken before this PR, so hopefully we can merge.